### ISSUE TITLE
v0.1.16: incorrect path generated 

### ISSUE THREAD

**Issue by florianduros** [jscarrott/mdbook-combiner#37]
Hi,

With the `v0.1.16`, the generated paths in the `SUMMARY.md` are incorrect. I don't face the issue with the `v0.1.15`.

I have the following folder structure:
```
- docs
    |-- project 
          | -- SUMMARY.md
          | -- README.md
```

The `SUMMARY.md` has a link to the `README.md`:
```md
- [Introduction](README.md)
```

Running the command:
`$ mdbook-combiner -m docs`

The generated `SUMMARY.md` has the following content:
```md
- [Introduction](README.md)
```

Instead to have an absolute path to the `README.md` like:
```md
- [Introduction](myabsolutepath/docs/project/README.md)
```

**Comment by jscarrott** [comment]
Thanks for the bug report, I'll have a look at this later today.

**Comment by jscarrott** [comment]
I think this should be fixed in 0.1.17
